adapters/mabidder/mabidder_test.go

```diff
@@ -0,0 +1,21 @@
+package mabidder
+
+import (
+	"testing"
+
+	"github.com/prebid/prebid-server/adapters/adapterstest"
+	"github.com/prebid/prebid-server/config"
+	"github.com/prebid/prebid-server/openrtb_ext"
+)
+
+func TestJsonSamples(t *testing.T) {
+	bidder, buildErr := Builder(openrtb_ext.BidderMabidder, config.Adapter{
+		Endpoint: "https://mabidder-test"},
+		config.Server{ExternalUrl: "http://hosturl.com", GvlID: 1, DataCenter: "2"})
```

Comment:
should use `https://prebid.ecdrsvc.com/pbs` url as mentioned in bidder-info param

https://github.com/prebid/prebid-server/blob/183345e49f0ff04bf9e6044dfa93b8d9ccffe9f3/static/bidder-info/mabidder.yaml#L1